### ISSUE TITLE
Extract image from opengraph not correct

### ISSUE THREAD

**Issue by rapsli** [goose3/goose3#45]
when extracting an image from open graph the src of the image is not fetched correctly. Here is the data I'm trying to fetch

```
<meta property="og:image" content="https://the-image.com/500x500.jpg" />
<meta property="og:image:secure_url" content="https://the-image.com/500x500.jpg" />
<meta property="og:image:width" content="500" />
<meta property="og:image:height" content="500" />
```

goose3 returns 500 when querying 

parsed_article.opengraph['image']

**Comment by rapsli** [comment]
Hi
Ok, here the correct markup:

```
<meta property="og:image" content="https://static.bienenwachstuch.ch/bienenwachstuch/wp-content/uploads/2018/01/27142239/Laundry-egg-Fragrance-Free-Egg-with-pellets-1-500x500.jpg" />
<meta property="og:image:secure_url" content="https://static.bienenwachstuch.ch/bienenwachstuch/wp-content/uploads/2018/01/27142239/Laundry-egg-Fragrance-Free-Egg-with-pellets-1-500x500.jpg" />
<meta property="og:image:width" content="500" />
<meta property="og:image:height" content="500" />
```

**Comment by barrust** [comment]
Thank you. Now I see what the issue you are describing is. The code to parse the opengraph data does the following to determine the key within the dictionary: 

``` python
value = self.parser.getAttribute(meta, 'content')
if value:
    opengraph_dict.update({attr.split(":")[1]: value})
```
Since it is splitting on `:` and using the second element, the key `image` is returning the last value: `500`. I think the best result would be to do something like this:

``` python
value = self.parser.getAttribute(meta, 'content')
if value:
    opengraph_dict.update({attr.split(":", 1)[1]: value})
```

Thoughts?

**Comment by rapsli** [comment]
that would be awesome! Will test it, but I'm not yet too familiar with Python and espacially with Goose

**Comment by barrust** [comment]
I will try to get something out in the next day or two. Working on school work at the moment but the change shouldn't be too hard to fix.

**Comment by rapsli** [comment]
no problem. let me know and I'll do some testing

**Comment by barrust** [comment]
@rapsli PR #46 should resolve this issue for you. Can you confirm that it is working of the master branch? I will get this pushed to pypi in the coming days. Thanks!

**Comment by rapsli** [comment]
@barrust It works as expected. Great job!

**Comment by barrust** [comment]
Awesome! This has been included in the 3.1.1 release which has been released to pypi.
